plexpy/notifiers.py

```diff
@@ -1676,3 +1719,240 @@ def return_config_options(self):
                          ]
 
         return config_option
+
+
+class Scripts(object):
+
+    def __init__(self, **kwargs):
+        pass
+
+    def conf(self, options):
+        return cherrypy.config['config'].get('Scripts', options)
+
+    def updateLibrary(self):
+        # For uniformity reasons not removed
+        return
+
+    def test(self, subject, message, *args, **kwargs):
+        self.notify(subject, message, *args, **kwargs)
+        return
+
+    def list_scripts(self):
+        scriptdir = plexpy.CONFIG.SCRIPTS_FOLDER
+        scripts = {'': ''}
+
+        if scriptdir and not os.path.exists(scriptdir):
+            os.makedirs(scriptdir)
```

Comment:
@Hellowlol We should raise an error, instead of trying to create the folder. (Permission issues) We can assume the user already decided where they put their scripts.